### PR TITLE
Fixed the overflowed signup screen

### PR DESCRIPTION
**Pull request by Samdid**
## Related Issue 

-Sign up screen is overflowing so there is need to fix it  

Closes: #68 

### Describe the changes you've made

I have fixed the overflowing sign up screen by adding Singlechildscrollview widget and reducing the height of textfields

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Mention any unusual behaviour of your code (Write NA if not)
Any unusual behaviour of your code

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Additional Info (optional)
Any additional information you want to give
Screenshot:
Screenshot_1617300018

### CODE REVIEW

File: Sheheroes_WatchOS/pubspec.lock [hunk 1]
```diff
@@ -0,0 +1,852 @@
+# Generated by pub
+# See https://dart.dev/tools/pub/glossary#lockfile
```
Comment (hiashutoshsingh) [inline Sheheroes_WatchOS/pubspec.lock:2]: please remove this file

File: pubspec.lock [hunk 2]
```diff
@@ -0,0 +1,1034 @@
+# Generated by pub
+# See https://dart.dev/tools/pub/glossary#lockfile
```
Comment (hiashutoshsingh) [inline pubspec.lock:2]: Please remove lock files from commit.

**Review by hiashutoshsingh — CHANGES_REQUESTED** [review]
Thanks for the PR. Just small changes(removing lock files) and then we are good to go.

File: pubspec.lock (hunk 2 above)
Comment (Samdid) [reply pubspec.lock]: Done

File: Sheheroes_WatchOS/pubspec.lock (hunk 1 above)
Comment (Samdid) [reply Sheheroes_WatchOS/pubspec.lock]: Done

**Comment by Samdid** [comment]
@hiashutoshsingh Changes resolved

**Review by hiashutoshsingh — APPROVED** [review]
Thanks for the contribution.

**Pull request merged**